store/store.go

```diff
@@ -213,6 +235,10 @@ func calcSeenCommitKey(height int64) []byte {
 	return []byte(fmt.Sprintf("SC:%v", height))
 }
 
+func calcBlockHashKey(hash []byte) []byte {
+	return []byte(fmt.Sprintf("H:%v", hash))
```

Comment:
```suggestion
	return []byte(fmt.Sprintf("BH:%v", hash))
```

prefix must be unique!